stable/parse/templates/NOTES.txt

```diff
@@ -78,8 +79,9 @@ service:
 
 {{- else if eq .Values.service.type "ClusterIP" }}
 
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/name={{ template "parse.name" . }},app.kubernetes.io/component=dashboard -o jsonpath="{.items[0].metadata.name}")
```

Comment:
The port-forward should be done to a service. Also, we should avoid to hardcode the ports.

Reply:
This chart is a little special. It's using a single service for both "server" and "dashboard" pods (playing with the common labels). 

This peculiarity is incompatible with `port-forwarding` to services.